### ISSUE TITLE
Alicorns/unicorns/pegasi

### ISSUE THREAD

**Issue by DracoMan671** [AlexModGuy/Ice-and-Fire-Suggestions#679]
what would a mythical creatures mod be without the mythical equine subspecies? i would love to see them added to the game as they would be extremely useful, [alicorns being the most useful] unicorns could help by defending you when you are attacked and pegasi would be able to fly you to locations, whereas Alicorns would have the benefits of both but would be much harder to find [maybe have an in-game event that takes place randomly on full moons and they only spawn in certain biomes]

**Comment by Mythtic12** [comment]
Have you tried this Mod? Wings Horns & Hooves, the Ultimate Unicorn Mod
this adds Unicorns, Alicorns and also Pegasus.. just like what you wanted.. But the bad thing is that the animations and weapons are little bit Kinda Lame.. so you idea might be Wonderful:)

**Comment by DracoMan671** [comment]
yeah. the part with the lame animations is kinda why i pretend that mod does not exist....

**Comment by Arasarhood** [comment]
> Have you tried this Mod? Wings Horns & Hooves, the Ultimate Unicorn Mod this adds Unicorns, Alicorns and also Pegasus.. just like what you wanted.. But the bad thing is that the animations and weapons are little bit Kinda Lame.. so you idea might be Wonderful:)

Wings, Horns and Hooves stops at 1.16.5 ... So, yes, please add Pegasus
